src/components/Pagination/Pagination.js

```diff
@@ -81,17 +81,20 @@ export const pagination = WrappedComponent =>
               />
             </Col>
             <Col className="text-right">
-              <select
-                value={this.state.page}
-                onChange={this.handlePageChange}
-                onBlur={this.handlePageChange}
-              >
-                {this.pagerCounter().map((value, i) => (
-                  <option value={value} key={i}>
-                    Page {value}
-                  </option>
-                ))}
-              </select>
+              <div className="select">
+                <span className="arr"></span>

```

Comment:
Why is this being added?

Reply:
As soon as the `select` element is styled, it loses its integrity so to speak. The arrow on the right side looks completely different, for example. Also, it seems the select element itself doesn't seem to be stylable nicely because its looks are decided by the browser if we don't override them. This is why the select class (for the button) and the arr class (the arrow on the right side) were added here. As a bonus, it looks more fitting to the whole theme.